### ISSUE TITLE
How to TIMIT trainAutoEncoder + TrainWithPretrain ?

### ISSUE THREAD

**Issue by Kaya122** [Microsoft/CNTK#626]
Dear all

I did without pre-trained auto encoder with TIMIT trainAutoencoder and TIMIT writeBottleneck configure, 
and now I would like to pre-trained auto encoder experiments.

I see train_withpretrained configure and I checked that they need to labels including align, and state.
However, I am doing autoencoder experiment in text-independent speaker verification system so I don't have any information about mlf and state. 

One question is that  can I omit the labels file in pretrain part?
If it is not possible, could you recommend any solution for me?

and for pretrain with autoencode, 
Is it okay to just change the TIMIT_Train3 to TrainAutoencoder part?

I mean 
command=TIMIT_DiscrimPreTrain1:TIMIT_AddLayer2:TIMIT_DiscrimPreTrain2:TIMIT_AddLayer3:TIMIT_TrainAutoEncoder
and add TIMT_TrainAutoEncoder part in pre-trained configure.

**Comment by wolfma61** [comment]
This issue has been dormant for a long while, please check out the latest CNTK release. If you still run into problems, please open a new issue.
